ale_linters/php/phpmd.vim

```diff
@@ -6,9 +6,18 @@ let g:ale_php_phpmd_executable = get(g:, 'ale_php_phpmd_executable', 'phpmd')
 " Set to change the ruleset
 let g:ale_php_phpmd_ruleset = get(g:, 'ale_php_phpmd_ruleset', 'cleancode,codesize,controversial,design,naming,unusedcode')
 
+" Set to tell phpmd files with which suffixes to check
+let g:ale_php_phpmd_suffixes = get(g:, 'ale_php_phpmd_suffixes', '')
+
 function! ale_linters#php#phpmd#GetCommand(buffer) abort
+    let l:suffixes = ale#Var(a:buffer, 'php_phpmd_suffixes')
+    let l:suffixes_option = !empty(l:suffixes)
+    \   ? ' --suffixes ' . l:suffixes
+    \   : ''
```

Comment:
I wonder, could we set this automatically? We could grab the file extension from the filename and pass that to the command, and then add tests for a few examples. Then you wouldn't need to configure anything for it to work. `expand('#' . a:buffer . ':e')` will get you the file extension without a dot, or an empty string if the file has no file extension.

Reply:
Good point. But what if user wants to specify an exact set of file extensions that phpmd should work with? Can it be done with ALE built-in functionality?

What do you think of following approach: we could use your solution when suffixes are not specified in `php_phpmd_suffixes` variable. Otherwise, only lint files with given extensions. It may be a bit overcomplicated, but then we will keep ability to prevent linting of files with certain extensions and user will not have to set up anything initially to make phpmd work

Reply:
Yeah, I think that could be a good option. The the variable is an empty string, default to using the current file extension, if there is one. Otherwise use the extensions from the variable.